Source/error.cpp

```diff
@@ -61,7 +61,7 @@ void InitDiabloMsg(char e)
 			return;
 	}
 
-	msgtable[msgcnt] = e;
+	msgtable[msgcnt] = e; // BUGFIX: this can cause an OOB

```

Comment:
```suggestion
	msgtable[msgcnt] = e; // BUGFIX: missing out-of-bounds check
```

Reply:
since it relies on hardcoded data, if you cause OOB, you kind of earned it lol